Lil-Peep/src/styles/modules/related-products.scss

```diff
@@ -0,0 +1,73 @@
+.related-product-container {
+  background-color: #{$product-card-category};
+  padding-left: 30px;
+  padding-right: 30px;
+
+  .collection-container {
+    background-color: #{$product-card-category};
+
+    .product-image {
+      justify-content: space-between;
+
+
+      a {
+        &:nth-of-type(odd) {
+          @media #{$breakpoint-tablet} {
+            display: none;
+          }
+        }
+      }
+
+      @media #{$breakpoint-tablet} {
+        justify-content: center;
+      }
+    }
+
+    .product-card-category {
+
+      &:last-child {
+
+        @media only screen and (max-width : 1139px) and (min-width: 769px) {
+          display: none;
+        }
+      }
+
+      &:nth-last-child(2) {
+
+        @media only screen and (max-width : 869px) {
+          display: none;
+        }
+      }
+
+      @media #{$breakpoint-mobile} {
+        max-width: none;
+      }
+    }
+
+    @media #{$breakpoint-tablet} {
+      background-color: #{$background-color};
+    }
+  }
+
+  @media #{$breakpoint-tablet} {
+    background-color: #{$background-color};
+  }
+}
+
+.related-product-title {
+  color: #{$title-font-gray};
```

Comment:
This also works like `color: $title-font-gray;`, it's cleaner and easier to read.

Reply:
really?! I did know that! I feel cheated.